src/flow.ts

```diff
@@ -612,7 +612,7 @@ export class Flow {
   /** Tests if the specified `this` field has the specified flag or flags. */
   isThisFieldFlag(field: Field, flag: FieldFlags): bool {
     var fieldFlags = this.thisFieldFlags;
-    if (fieldFlags) {
+    if (fieldFlags != null && fieldFlags.has(field)) {
```

Comment:
This is old code apparently, written before the requirement for a `Map#has` before a `Map#get` was conceived. Produced a wholesome key-does-not-exist error in Wasm.